tests/core-draw-target-cropped.rs

```diff
@@ -0,0 +1,138 @@
+use embedded_graphics::{
```

Comment:
Why did you move these tests to here?

Reply:
IIRC it was to keep the e-g dependency out of core. It should be possible to add it as a dev dependency and move these tests back into their respective files, but I didn't do that in case it causes more dep conflicts in the future. What would you suggest?

Reply:
But doesn't the core crate already have a dev-dependency on e-g?

Reply:
Yes it does :facepalm:. Not sure where I was going with the tests moved here, but I can move them back. With one strange quirk: all their dependencies must come from e-g to compile. I can't use a mix of `crate::` and `embedded_graphics::`. Unsure why, but they work now so :shrug:

Reply:
I think this is be caused by the cyclic dev-dependency. Cargo will have to break the cycle somewhere and this makes the types from `crate` and `embedded_graphics` incompatible. 

The cyclic dev-dependency will also cause a problem when we want to publish the crate, because we can't publish one create without the other already existing. But this can be fixed by removing the `version` from the dev-dependency in the core crate: See the note at https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies

Reply:
Another TIL on Cargo's behaviour. Is it just me or is it really complex when you get into the weeds? Anyway, I've removed the version so now the dev dependency should hopefully not cause any problems.